### PR TITLE
replaced detached configuration with explicit tooling configs so that customizations via resolution strategy and spring dependency plugin are possible

### PR DESCRIPTION
**Pull request by cdietrich**
replaced detached configuration with explicit tooling configs so that customizations via resolution strategy and spring dependency plugin are possible

Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,4 +7,4 @@ A set of [Gradle](http://gradle.org) plugins to build and use [Xtext](http://xte
 
 The documentation is hosted on the [project's website](http://xtext.github.io/xtext-gradle-plugin/).
 
-To build and install locally run `./gradlew install -Dorg.gradle.project.version=<version>` e.g. with `<version>` set to `1.0.22-SNAPSHOT`.
+To build and install locally run ` ./gradlew clean install -Dorg.gradle.project.version=<version> -Dmaven.repo.local=.m2` e.g. with `<version>` set to `1.0.22-SNAPSHOT`.
```
Comment (oehme) [inline README.md]: Can you revert this please? You don't need to run clean with Gradle and the custom m2 will break the tests.

File: xtext-gradle-plugin/src/main/java/org/xtext/gradle/tasks/XtextSourceDirectorySet.xtend [hunk 2]
```diff
@@ -27,4 +27,12 @@ interface XtextSourceDirectorySet extends PatternFilterable {
 	def void output(Closure<?> configureClosure)
 
 	def String getGeneratorTaskName()
+
+	def String getXtextToolingConfigurationName() {
+		getXtextToolingConfigurationName("xtextTooling")
+	}
+
+	def String getXtextToolingConfigurationName(String prefix) {
```
Comment (oehme) [inline xtext-gradle-plugin/src/main/java/org/xtext/gradle/tasks/XtextSourceDirectorySet.xtend]: How about we only keep this method and call it something like "qualifyConfigurationName"?

Comment (cdietrich) [reply xtext-gradle-plugin/src/main/java/org/xtext/gradle/tasks/XtextSourceDirectorySet.xtend]: will change later today

File: README.md (hunk 1 above)
Comment (cdietrich) [reply README.md]: checked

**Pull request merged**